ui/segmented-bar.reel/segmented-bar.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @module montage/ui/segmented-bar.reel
+ * @requires montage/ui/base/abstract-button.AbstractButton
+ * @requires montage/core/tree-controller.TreeController
+ */
+var AbstractButton = require("ui/base/abstract-button").AbstractButton;
+
+/**
+ * @class SegmentedBar
+ * @classdesc
+ * A segmented bar.
+ * Reference design: OS X storage space visualization found in iTunes or About This Mac
+ * Inheriting from AbstractButton because SegmentedBar needs its PressComposer behavior
+ *
+ * @extends AbstractButton
+ *
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes}
+ * for FlexBox usage
+ */
+exports.SegmentedBar = AbstractButton.specialize( /** @lends SegmentedBar.prototype # */ {
```

Comment:
Actually I don't know why there's a `#` at all, they're all over the place in the codebase, we need to fix it. 

Though it's crucial to have `.prototype` because they're instance methods/properties, not static.